.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Types of changes
```

Comment:
Would probably make sense to also require a reference to a specific issues and, per our postmortem discussion, have something like "priority" to communicate how urgent a change is.